extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -189,7 +197,18 @@ function getPlacementFromObject(win, placementObj) {
     dev().warn(TAG, 'Parentless anchor with BEFORE/AFTER position.');
     return null;
   }
-  return new Placement(win, anchorElement, injector);
+  let margins = undefined;
+  if (placementObj['style']) {
+    const marginTop = parseInt(placementObj['style']['top_m'], 10);
+    const marginBottom = parseInt(placementObj['style']['bot_m'], 10);
+    if (marginTop) {
+      margins = {top: marginTop};
+    }
+    if (marginBottom) {
+      (margins = margins || {}).bottom = marginBottom;
```

Comment:
If we set the margins to anything (including 0), then it means the attemptSizeChange logic has to read the existing margins using getComputedStyle (which is expensive), to figure out what the diff is. So if we're not setting any margins it's best to just provide undefined I think.